src/app/app.module.ts

```diff
@@ -158,12 +166,16 @@ const routes: Routes = [
     MessageListService,
     MobileQueryService,
     RunboxWebmailAPI,
+    RMM,
     RMMAuthGuardService,
     StorageService,
     { provide: HTTP_INTERCEPTORS, useClass: RMMHttpInterceptorService, multi: true}
   ],
   bootstrap: [MainContainerComponent],
-  entryComponents: [MoveMessageDialogComponent]
+  entryComponents: [MoveMessageDialogComponent,
+    ProfilesEditorModal,
```

Comment:
Shouldn't this be in the identities module? No other (non webmail related) dialog is specified here in the app.module.ts.

Reply:
Tried to remove this and added in profiles.modue.ts however the modal does not open.
I think it needs to be here.

Reply:
That's because your ProfilesModule is not actually being loaded into the app module. I have no idea how or why it even works in the first place: it did not load for me in the `npm run start` mode because of the unloaded module. I fixed it [here](https://github.com/runbox/runbox7/commit/00a2fdfdfbf1de875303548375fd5c6de4659778) – now it also passes the tests :)